### DISCUSSION TITLE
Actividad 1 - GRUPO 13

### DISCUSSION THREAD

**Discussion by lucasmartinez22** [frlp-utn-ingsoft/catedra#95] [Show and tell]
https://github.com/facularrauri/vetsoft.git

**Comment by RodrigoJacznik** [comment]
Buenas!
La actividad está bien resuelta, queda aprobada. Siguieron muy bien feature branch.

Intenten normalizar los nombres de los branch (por ejemplo medicine no sigue el formato que sigue el resto)

Se pueden mejorar los mensajes de los commits.
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/58c25892-eb64-4ce7-9c11-d81da1ae2f20)
Por ejemplo:

- 11c688a y 3990834 son muy genéricos
- 20365ba nombra innecesariamente el archivo donde se realiza el cambio

Saludos
